README.md

```diff
@@ -235,7 +235,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
     * [coala](http://coala-analyzer.org/) - Language independent and easily extendable code analysis application.
     * [code2flow](https://github.com/scottrogowski/code2flow) - Turn your Python and JavaScript code into DOT flowcharts.
     * [pycallgraph](https://github.com/gak/pycallgraph) - A library that visualises the flow (call graph) of your Python application.
-    * [pysonar2](https://github.com/yinwang0/pysonar2) - A type inferencer and indexer for Python.
+    * [pysonar2](https://github.com/DreamLinuxer/pysonar2) - A type inferencer and indexer for Python.
```

Comment:
Yeah I speak Chinese and it says he no longer does open source because he is tired of people repackaging his inventions and selling them. However, he says that the old code is still open, it's only new versions that will be closed source - so other people's clones of his repositories are still legal/available for use. I think it would be a good idea to find clones of the software and link to those - if that sounds good to you let me know and find those and I'll update this PR.